src/index.js

```diff
@@ -0,0 +1,80 @@
+// Imported Depndencies
+const fs = require("fs");
+const { prompt } = require("inquirer");
+
+//  Imported Classes
+const Engineer = require("./lib/Engineer");
+const Manager = require("./lib/Manager");
+const Intern = require("./lib/Intern");
+
+// Imported Questions
+const {
+    teamNameQuestions,
+    managerQuestions,
+    optionQuestions,
+    engineerQuestions,
+    internQuestions,
+} = require("./questions");
+
+// Imported HTML
+const htmlTemplate = require("./html");
+
+// Write HTML File
+const writeToFile = (filePath, data) => {
+    try {
+        fs.writeFileSync(filePath, data);
+    } catch (error) {
+        console.log(error.message);
+    }
+};
+
+// Call this function to start application
+const start = async() => {
+    // store employee objects here
+    const employeeArray = [];
+
+    // ask team name question
+    const teamNameAnswers = await prompt(teamNameQuestions);
```

Comment:
Destructure `{ teamName }` directly here, then on line 74 we can just pass `teamName` instead of `teamNameAnswers.teamName`